.github/issue_template.md

```diff
@@ -0,0 +1,22 @@
+PLEASE FOLLOW THE CHECKLIST BELOW WHEN CREATING A NEW ISSUE. THE CHECKLIST IS
+FOR YOUR INFORMATION AN MUST BE REMOVED BEFORE SUBMITTING ISSUE.
+
+## Checklist
+
+- [ ] Is the issue marked with exactly one of the following labels
+
+    - Bug
+    - Feature
+    - Improvement
+    - infrastructure
```

Comment:
We don't actually need this anymore. Shouldn't be in the checklist in my opinion.

Reply:
Ah this is for the issues. Ok then.